env/_template

```diff
@@ -0,0 +1,14 @@
+LISK_CORE_URL="http://localhost:4000"
```

Comment:
Quotes will be interpreted litteraly in this file: I think you want to remove them.